ksql-api/src/main/java/io/confluent/ksql/api/server/ErrorCodes.java

```diff
@@ -23,11 +23,12 @@
   private ErrorCodes() {
   }
 
-  public static final int ERROR_CODE_MISSING_PARAM = 50001;
```

Comment:
Why the decision to remove the `500` prefixes? These error codes are user-facing, right?

Reply:
I don't think we should tie our error codes to HTTP status codes, they seem orthogonal. 
E.g. If I stream some inserts to the server, then I'll get a 200 OK response and the acks for the inserts will start coming up. Maybe the 100000th insert has some malformed JSON in which case the ack stream will contain an error "malformed JSON" and the response will be ended. The error here has got nothing to do with the HTP status code (which was 200 in this case).

Reply:
These error codes are exposed to users, though, right? How will a user understand the meaning of the error codes? (Or is that not the purpose of the error codes?)